src/main/java/net/snowflake/client/jdbc/SnowflakeColumnMetadata.java

```diff
@@ -212,4 +212,8 @@ public String toString() {
 
     return sBuilder.toString();
   }
+
+  public FieldMetadata getField(int index) {
```

Comment:
`columnIndex`? And to be honest - if we cant to keep code cleaner it would be great to have some class wrapping list of field metadata and hiding this index remapping. But not in this PR definitely.

Reply:
Maybe fieldIndex or simply index. The field could be inside structure type and then it's not a column.

Reply:
In the end the method was removed